spec/models/post_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+describe Post, type: :model do
+  test_user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.',
+                          post_counter: 0)
+  test_post = Post.create(author: test_user, title: 'Hello', text: 'This is my first post', comments_counter: 0,
+                          likes_counter: 0)
+
+  before { test_post.save }
+  it 'should have a title' do
+    expect(test_post.title).to eq 'Hello'
+  end
```

Comment:
- [ ] @Mutai-Gilbert great work adding a test for the title, however, you forgot to add a test to validate the title only accepts a title with 250 characters or less.

Reply:
- [ ] @Mutai-Gilbert great work adding tests for the post model, however, you forgot to add tests for the `increment_post_counter` custom method In post model.

Reply:
Done

Reply:
Done.

Reply:
Done